docs/src/content/docs/guides/css-and-tailwind.mdx

```diff
@@ -182,7 +182,7 @@ If you already have a Starlight site and want to add Tailwind CSS, follow these
     @import 'tailwindcss/utilities.css' layer(utilities);
     ```
 
-4.  Update the Starlight config to use the Tailwind CSS file:
+4.  Update the Starlight config to add the Tailwind CSS file at the first position of the `customCss` array:
```

Comment:
There are probably a couple of alternative phrasings here that might be better:

```suggestion
4.  Update the Starlight config to add the Tailwind CSS file as the first item in the `customCss` array:
```

Reply:
or
```suggestion
4.  Update the Starlight config to add the Tailwind CSS file to the start of the `customCss` array:
```
I think either of these could work potentially.

Reply:
Thanks a lot, definitely reads better now (picked the first one as I like that there is no possible confusion with the word "first") :raised_hands: